Sources/SPMTestSupport/XCTAssertHelpers.swift

```diff
@@ -94,3 +95,17 @@ public func XCTAssertBuildFails(
         XCTFail("`swift build' failed in an unexpected manner")
     }
 }
+
+public func XCTAssertEqual<T: CustomStringConvertible>(
+    _ assignment: [(container: T, version: Version)],
+    _ expected: [T: Version],
+    file: StaticString = #file, line: UInt = #line
```

Comment:
In the interest of keeping this focused, let's punt on that. For something like `#file` usage, I think we'll want to change everything project-wide at once, rather than adopting piecemeal over time.

Reply:
Sounds good.